docs/reference/commandline/plugin_set.md

```diff
@@ -34,6 +34,48 @@ The settings currently supported are:
  * path of devices
  * args
 
+## What is settable ?
+
+Look at the plugin manifest, it's easy to see what fields are settable,
+by looking at the `Settable` field.
+
+Here is an extract of a plugin manifest:
+
+```
+{
+        "Config": {
+            ...
+            "Env": [
+                {
+                    "Name": "DEBUG",
+                    "Settable": ["value"],
+                    "Value": "0"
+                },
+                {
+                    "Name": "LOGGING",
+                    "Value": "1"
+                }
+            ],
+	    "Mounts": [
+                {
+                    "Destination": "/mnt/state",
+                    "Name": "state",
+                    "Options": ["rbind"],
+                    "Settable": ["source"],
+                    "Source": "/var/lib/docker/plugins/",
+                    "Type": "bind"
```

Comment:
Can you also add example sections for `Args` and `Devices` and explain them for completeness? w.r.t `Args`, please use a Name that's not `Args` :)